src/cmd/src/frontend.rs

```diff
@@ -216,6 +216,7 @@ impl StartCommand {
 
         if let Some(addr) = &self.rpc_addr {
             opts.grpc.addr.clone_from(addr);
+            opts.grpc.tls = tls_opts.clone();
```

Comment:
nit: using the same certificate for multiple services is not the best security practice. However, it's acceptable for this PR to keep it simple; we can refactor it in the future.

Reply:
Okay. For now, it seems  that`grpc`,`mysql`,`pg` are all using the same TLS configuration directly. We could refactor this: reading the TLS configuration from  config file firstly, or retaining the original logic.